presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -4870,6 +4870,20 @@ public void testExecuteUsingWithSubquery()
                 "SELECT 10 in (SELECT orderkey FROM orders)");
     }
 
+    @Test
+    public void testExecuteWithParametersInLambda()
+    {
+        String query = "SELECT filter(array[1, 2, 3], x -> x > ?)";
+        Session session = Session.builder(getSession())
+                .addPreparedStatement("my_query", query)
+                .build();
+
+        assertQuery(
+                session,
+                "EXECUTE my_query USING 2",
+                "SELECT array[3]");
```

Comment:
It's a very short function call. We can put everything in one line. BTW, java has super long line chars (150 - 200 per line is ok)

Reply:
Done. Thanks for the suggestion.